tests/e2e/test_rosacli_cluster_post.go

```diff
@@ -585,7 +586,10 @@ var _ = Describe("Post-Check testing for cluster deletion",
 				clusterDetail, err := profilehandler.ParserClusterDetail()
 				Expect(err).To(BeNil())
 				oidcEndpointUrlC = clusterDetail.OIDCEndpointURL
-				oidcEndpointUrlC, err = common.ExtractOIDCProviderFromOidcUrl(oidcEndpointUrlC)
+				parsedUrl, err := url.Parse(oidcEndpointUrlC)
+				Expect(err).To(BeNil())
+				oidcEndpointUrl := parsedUrl.String()
+				oidcEndpointUrlC, err = common.ExtractOIDCProviderFromOidcUrl(oidcEndpointUrl)
```

Comment:
I do not realize the function `ExtractOIDCProviderFromOidcUrl,` but it seems like we are in-cycle to deal with `oidcEndpointUrlC.` 

What does the origin value of oidcEndpointUrlC like? and what kind of final value do you want?

Reply:
The Original value of `oidcEndpointUrlC` is like `https://akanni-joyjne0s-oidc-y2k4.s3.us-west-2.amazonaws.com (Unmanaged)` so when we try to get `oidcProvider` using `ExtractOIDCProviderFromOidcUrl`, it is throwing an error because the above `oidcEndpointUrlC` value has white space and (Unmanaged) after the actual URL. 

I guess changing 
`oidcEndpointUrlC, err = common.ExtractOIDCProviderFromOidcUrl(oidcEndpointUrl) `
this to 
`oidcProvider, err := common.ExtractOIDCProviderFromOidcUrl(oidcEndpointUrl)`
will make it more readable

Reply:
Just now realized, assigning new variable `oidcProvider` also requires to use it in code, as we are not performing any actions on the output value i will use _ instead of variable `oidcProvider`.